### DISCUSSION TITLE
Updating FTC

### DISCUSSION THREAD

**Discussion by Thebasestar** [gekkekoe/esphome-ecodan-hp#96] [General]
Hi Gekkekoe,
found ftc firmware update here: https://drive.google.com/drive/folders/18RT9mcwQj4Ro-d09x-T1mGcMWAl2tR2e , it came from this forum where you look present (different user name) https://gathering.tweakers.net/forum/list_messages/2112892/0 .
Did you update your? I'm actually on 17.00 Is it worth it?

Thanks

**Comment by DamijanD** [comment]
Is this the official firmware or some modification? And are there any changelogs?

**Comment by gekkekoe** [comment]
It's the original firmware and I have not updated yet. I had an issue with my unit and was waiting for that to be sorted out before considering updating.

There are no changelogs available, but users report less defrosting and lower frequencies. But probably depends on what firmware you had before 

But in general, it it ain't broke, don't fix it

**Comment by gekkekoe** [comment]
So I had some issues with energy consumption and bugs for 29 feb. Also wanted to test the better defrost, so I did the upgrade ;-)

**Comment by merlin051** [comment]
Any noticeable difference?

**Comment by gekkekoe** [comment]
way to early to tell, But If I notice any, will document them here

[edit]
* DHW eco mode seems to be limited at 70hz instead of 60hz
